pytest.ini

```diff
@@ -0,0 +1,13 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
+norecursedirs = .svn _build ndarray* # ignore tests in ndarray due to missing stdVector module not found
+testpaths =
+    tests
+python_files = *_TestCase.py histogramTest_*.py #manually run python alltests.py will also return 145 tests
```

Comment:
does this include all the test files? For example are these:
- test_matplotlib_contourf.py
- test_matplotlib_pcolor.py
executed through pytest? Are there any other tests that need to be included, too?

Reply:
These two were not included in the alltests.py. If you look into the body of these two functions, they have no unittest class in it. It's simply generating a plot that has nothing to do with Histogram package. Not sure if it's needed to add into pytest. 

A note is histogram use pylab through out its code, but pylab is getting deprecated by pyplot. So this may create problems down the road. Currenlty it's not a problem yet.